packages/components-vue/src/examples/SelectExample.vue

```diff
@@ -0,0 +1,26 @@
+<script setup lang="ts">
+  import { PSelect, PSelectOption } from '@porsche-design-system/components-vue';
+  import { ref } from 'vue';
+
+  const lastSubmittedData = ref('none');
+  const handleSubmit = (e: Event) => {
+    const formData = new FormData(e.target as HTMLFormElement);
+    lastSubmittedData.value = formData.get('options')?.toString() || 'none';
+  };
+</script>
+
+<template>
+  <form @submit.prevent="handleSubmit">
+    <PSelect :name="'options'" :label="'Some Label'" value="'a'">
```

Comment:
I guess copy/paste from multi select but here it is just singular. 
```suggestion
    <PSelect :name="'option'" :label="'Some Label'" value="'a'">
```

Reply:
I guess `options` is also fine for a select name where you have multiple options to choose from